plutus-pab/src/Cardano/Wallet/Mock/API.hs

```diff
@@ -36,7 +38,9 @@ PSGenerator we specialise it to 'Text'.
 type API walletId -- see note [WalletID type in wallet API]
     = "create" :> QueryParam "funds" Integer :> Post '[JSON] WalletInfo
       :<|> Capture "walletId" walletId :> "submit-txn" :> ReqBody '[JSON] CardanoTx :> Post '[JSON] NoContent
-      :<|> Capture "walletId" walletId :> "own-payment-public-key" :> Get '[JSON] WalletInfo
+      -- TODO: Should we removed in favor of 'own-addresses'. However, how do we deprecate an HTTP request?
```

Comment:
I have found the post on this topic https://medium.com/co-star-engineering/deprecating-api-endpoints-with-servant-1c00f8a211ca, but it was not added to the servant itself yet — https://github.com/haskell-servant/servant/issues/1509

I think this endpoint doesn't worth the efforts. But we may deprecate the client function `Cardano.Wallet.Mock.Client.ownPaymentPubKeyHash`.

Reply:
Good point! I'll deprecate the client function.